modules/proc_manage-clairv4.adoc

```diff
@@ -290,12 +266,60 @@ with the updated configuration to be deployed.
 At this point, images in any of the organizations identified in the
 namespace whitelist will be scanned by Clair v4.
 
-== Using Clair v4
-The user interface for viewing vulnerability information gathered by Clair v4
-is essentially the same as it was for Clair v2.
+== Setting up Clair on a non-OpenShift {productname} deployment
+
+For {productname} deployments not running on OpenShift, it is possible to configure Clair security scanning.
+
+. Deloy a (preferably fault-tolerant) Postgres database server.  Note that Clair requires the `uuid-ossp` extension to be added to its Postgres database.  If the user supplied in Clair's `config.yaml` has the necessary privileges to create the extension then it will be added automatically by Clair itself.  If not, then the extension must be added before starting Clair.  If the extension is not present, the following error will be displayed when Clair attempts to start.
++
+```
+ERROR: Please load the "uuid-ossp" extension. (SQLSTATE 42501)
+```
++
+. Create a Clair config file in a specific folder (e.g. `/etc/clairv4/config/config.yaml`) file.
++
+.config.yaml
```

Comment:
do you want to link to config https://quay.github.io/clair/reference/config.html
We will keep that link up to date with config information.